src/stationd/amplifier.py

```diff
@@ -1,263 +1,243 @@
 import time
-from multiprocessing import Manager
-from typing import TYPE_CHECKING
-
-if TYPE_CHECKING:
-    from multiprocessing.managers import DictProxy
 
 from . import stationd as sd
-from .gpio.gpio import GPIOPin
+from .gpio.gpio import HIGH, LOW, GPIOPin
+
+MOLLY_TIME = 20  # In seconds
+PTT_COOLDOWN = 120  # In seconds
+SLEEP_TIMER = 0.1
+
+LEFT = HIGH
+RIGHT = LOW
```

Comment:
(Naive) Question: Does `LEFT` and `RIGHT` here refer to polarization or does it refer to some other sense of directionality?  Why not just use gpio's `HIGH` and `LOW` constants?

Reply:
Excellent!  That's what I thought but wasn't 100% sure.  Maybe a comment but as you pointed it, it becomes clearer when you read where/how it's used.

I asked in part because it has implications for [my work as I rip out gpio submodule](https://github.com/uniclogs/uniclogs-stationd/pull/18).  ;)